packages/flutter_tools/test/general.shard/common_test.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../src/common.dart';
+
+void main() {
+  final Map<dynamic, dynamic> stubMatchState = <dynamic, dynamic>{};
```

Comment:
Even though this is trivial (and looks safe), we are trying to remove all state that is persisted between tests (it often leads to flakes or tests passing because they ran after a specific test which set up state).

I think you can re-write these tests to be one-liners, of the form:

`expect('Give me any text!', containsIgnoringWhitespace('any text!'));`

or

`expect('any text!', isNot(containsIgnoreWhitespace('Give me anytext!')));`

Reply:
done!